apps/chameleon/components/user-auth-form.tsx

```diff
@@ -30,6 +30,9 @@ export function UserAuthForm({ className, ...props }: UserAuthFormProps) {
   const [isLoading, setIsLoading] = React.useState<boolean>(false)
   const [isGitHubLoading, setIsGitHubLoading] = React.useState<boolean>(false)
   const searchParams = useSearchParams()
+  const variantType: string = props.styleVariant
```

Comment:
Bringing in props like this breaks the headless model.  If you want to extend the current you need to do so with consideration for the atomic model.